cura/CuraApplication.py

```diff
@@ -1635,15 +1637,14 @@ def _readMeshFinished(self, job):
                     node.scale(original_node.getScale())
 
             node.setSelectable(True)
-            node.setName(os.path.basename(filename))
+            node.setName(os.path.basename(file_name))
             self.getBuildVolume().checkBoundsAndUpdate(node)
 
             is_non_sliceable = False
-            filename_lower = filename.lower()
-            for extension in self._non_sliceable_extensions:
-                if filename_lower.endswith(extension):
-                    is_non_sliceable = True
-                    break
+
+            if file_extension in self._non_sliceable_extensions:
```

Comment:
This is actually not correct... the extensions in `self._non_sliceable_extensions` have a dot in the front such as ".gcode". Fixed this problem in 1f15599b4